backend/compact-connect/lambdas/nodejs/lib/event-client.ts

```diff
@@ -138,6 +138,41 @@ export class EventClient {
         return resp.Items?.map((item) => unmarshall(item) as IIngestFailureEventRecord) || [];
     }
 
+    /*
+     * Queries the data event table for ingest successes and returns the first 100 matching records.
+     *
+     * This is used to determine if a jurisdiction has had any successful uploads within the
+     * time window.
+     */
+    public async getIngestSuccesses(
+        compact: string,
+        jurisdiction: string,
+        startTimeStamp: number,
+        endTimeStamp: number
+    ): Promise<IIngestSuccessEventRecord[]> {
+        this.logger.info('Getting ingest failures', {
+            compact: compact,
+            jurisdiction: jurisdiction,
+            start_time_stamp: startTimeStamp,
+            end_time_stamp: endTimeStamp,
+        });
+        const resp = await this.dynamoDBClient.send(new QueryCommand({
+            TableName: environmentVariables.getDataEventTableName(),
+            Select: 'ALL_ATTRIBUTES',
+            // We don't necessarily return all the matching records for this check
+            // since we're just looking for the presence of any ingest successes
```

Comment:
If this function is used only to check that there is at least one ingest success (what I assume you mean by "this check"), why not set `limit: 1` and/or return a boolean instead of a list of records?

(If you prefer to keep it this way, can we update the comment and clarify what you mean by "this check"?)